app/assets/stylesheets/application.css.scss

```
@@ -3,6 +3,9 @@
 @import 'required';
 @import 'vendor';
 @import 'vendor-overrides';
+// To be removed once design system incorporates styles included below.
+@import '_design-system-waiting-room.scss';
```

Minor: For consistency with other imports here, we could omit the leading `_` and trailing file extension.

```suggestion
@import 'design-system-waiting-room';
```